Library/Homebrew/cmd/commands.rb

```diff
@@ -8,6 +8,14 @@ module Homebrew
   module_function
 
   def commands
+    require "options_for_brew_commands"
+    options do
+      command __method__.to_s
```

Comment:
```__method__.to_s``` can also be passed as an argument to ```options```, like:
```
options(__method__.to_s) do
...
end
```

Let me know which one you prefer.